test/examples/full-form.json

```diff
@@ -61,7 +61,7 @@
             },
             {
               "label": "Password",
-              "required": false,
+              "required": true,
```

Comment:
Based on Filippe's recollection, the admin user is not used internally. So with sso/ldap, it should not be required at all. That means we can set it back to false, and in fact it would be ignored with sso/ldap, which is fine.

@ruromero please investigate if you are skeptical of this being true :-)